projects/plugins/search/src/class-jetpack-search-plugin.php

```diff
@@ -56,4 +63,32 @@ public static function initialize() {
 		// Initialize My Jetpack.
 		My_Jetpack_Initializer::init();
 	}
+
+	/**
+	 * Activation hook.
+	 */
+	public static function handle_plugin_activation() {
+		// If site is already connected, enable the search module and enable instant search.
+		if ( ( new Connection_Manager() )->is_connected() ) {
+			$controller        = new Search_Module_Control();
+			$activation_result = $controller->activate();
+
+			if ( ! is_wp_error( $activation_result ) && true === $activation_result ) {
```

Comment:
```suggestion
			if ( true === $activation_result ) {
```
Should we just leave this condition if I didn't get it wrong? because an error wouldn't `===` true.

Reply:
Good point, fixed!